bolt/src/main/java/com/slack/api/bolt/service/builtin/oauth/OpenIDConnectErrorHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.slack.api.bolt.service.builtin.oauth;
+
+import com.slack.api.bolt.request.builtin.OAuthCallbackRequest;
+import com.slack.api.bolt.response.Response;
+import com.slack.api.methods.response.openid.connect.OpenIDConnectTokenResponse;
+
+/**
+ * An error handler for errors with `oauth.v2.access` API calls.
```

Comment:
Is this class for the oauth.v2.access API endpoint? I thought it was for openid.connect?

Reply:
Good catch! Thanks!

Reply:
Fixed in the latest commit